src/main/java/pl/wedding/management/service/weddinghall/model/AddressHall.java

```diff
@@ -0,0 +1,21 @@
+package pl.wedding.management.service.weddinghall.model;
+
+import lombok.*;
+
+@Builder
+@Getter
+public class AddressHall {
+    private final String country;
+    private final String city;
+    private final String street;
+    private final int numberBuilding;
+    private final String postCode;
+
+    public AddressHall(String country, String city, String street, int numberBuilding, String postCode) {
```

Comment:
Why not lombok `@RequiredArgsConstructor` , you already use `@Getter` and `@Builder`

Reply:
jesli zamiast konstruktora dam @RequiredArgsConstructor podkresla mi pola na czerwono

Reply:
ok, don't add it. Resolved :)